apix-rest-v1/src/main/java/eu/xenit/apix/rest/v1/nodes/NodesWebscript1.java

```diff
@@ -744,4 +797,12 @@ public Object execute() throws Throwable {
                 .nodeRefToNodeInfo((NodeRef) resultRef, fileFolderService, nodeService, permissionService);
         writeJsonResponse(response, nodeInfo);
     }
+
+    @ExceptionHandler(AccessDeniedException.class)
+    private void writeNotAuthorizedResponse(AccessDeniedException exception, WebScriptResponse response)
+            throws IOException {
```

Comment:
The writeJsonResponse method can throw IOExceptions because it uses the jackson objectmapper which can throw those exceptions. I am not sure how we would have to handle that.

Reply:
This writeJsonResponse method is used in many places, even in other places where other exception are caught. The writeJsonResponse method is a method that is implemented in the ApixV1Webscript class. I think that if we want to get rid of that potential IOException that we should fix it in that method. That is out of scope for this ticket I would say.